azure_jumpstart_arcbox/artifacts/DataOpsLogonScript.ps1

```diff
@@ -257,10 +258,14 @@ while ($(Get-Job -Name arcbox).State -eq 'Running') {
 Get-Job -name arcbox | Remove-Job
 write-host "Successfully deployed Azure Arc Data Controllers"
 
+Stop-Transcript
```

Comment:
is this meant to stop the transcript started in the for loop? if so, shouldnt we stop it in the for loop instead of after?

Reply:
removed  this one